src/templatizer.js

```diff
@@ -3,7 +3,21 @@ const templatizer = (options) => {
     const optionTemplate = require('.template/option.json');
 
     let result = argumentsTemplate;
-    //TODO: create implementation of templatizer function
+    options.forEach(function (option) {
+        let option_schema = optionTemplate;
```

Comment:
If you use `camelCase` for variables - please use it everywhere.

Reply:
And also pay attention that this is a reference to the variable that you change several times during `forEach`. I expect that you will have an array of elements equal to the last element.

Reply:
Example: https://jsbin.com/rimofawigo/edit?js,console